plugins/modules/external_usergroup.py

```diff
@@ -97,7 +107,15 @@ def main():
                 entity = external_usergroup
 
         module.set_entity('entity', entity)
-        module.auto_lookup_entities()
+
+        auth_source = module.lookup_entity('auth_source')
```

Comment:
this feels more natural than doing `auto_lookup_entities` and then `pop`ing it out of `foreman_params`.

interestingly not `pop`ing it does not affect idempotency